commit 65d845f891a3
Author: Jet Villegas <jvillegas@mozilla.com>
Date:   Tue Jan 9 08:00:00 2018 +0200

    Bug 1426869 - td.contenteditable=true focus breaks text selection. r=masayuki, a=RyanVM
    
    --HG--
    extra : source : b3118b4fae1372c3aa57caf4b16af9834ce46539
    extra : amend_source : 1187aa7d3d760aa60b974b874968aea4d0e09eea
---
 layout/generic/nsFrameSelection.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git layout/generic/nsFrameSelection.cpp layout/generic/nsFrameSelection.cpp
index c483622e1ffb..a60d69bf79e0 100644
--- layout/generic/nsFrameSelection.cpp
+++ layout/generic/nsFrameSelection.cpp
@@ -1403,6 +1403,7 @@ nsFrameSelection::TakeFocus(nsIContent*        aNewFocus,
 
     NS_ENSURE_STATE(mShell);
     bool editableCell = false;
+    mCellParent = nullptr;
     RefPtr<nsPresContext> context = mShell->GetPresContext();
     if (context) {
       RefPtr<HTMLEditor> htmlEditor = nsContentUtils::GetHTMLEditor(context);
